package.json

```diff
@@ -19,7 +19,10 @@
     "mocha": "^10",
     "sinon": "^15"
   },
-  "exports": "./src/index.js",
+  "exports": {
+    ".": "./src/browser/index.js",
+    "./wtr-config.js": "./src/server/index.js"
+  },
```

Comment:
Had to split the exports since having them all in one file was causing the browser to blow up on Node.js things.